v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/source/reader/io/row/SourceRowTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.source.reader.io.row;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.teleport.v2.source.reader.io.schema.SchemaTestUtils;
+import junit.framework.TestCase;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/** Test class for {@link SourceRow}. */
+@RunWith(MockitoJUnitRunner.class)
+public class SourceRowTest extends TestCase {
+  @Test
+  public void testSourceRowBuilds() {
+    final String testTableUUID = "fa14a160-2841-42f2-91ee-7929d791f367";
```

Comment:
nit: The testTableUUID can be removed from the tests in all the files.

Reply:
Thanks. done.